w3d_translator/errors.py

```diff
@@ -0,0 +1,14 @@
+class ValidationError(Exception):
+    def __init__(self, message):
+        self.message = message
+
+    def __str__(self):
+        return self.message
```

Comment:
Pretty sure you don't need `__init__` or `__str__` here:

```
>>> class MyError(Exception):
...     def __init__(self, message):
...             self.message = message
...
>>> MyError('hello')
MyError('hello')
>>> str(MyError('hello'))
'hello'
>>> class MyError2(Exception):
...     pass
...
>>> MyError2('hello world!')
MyError2('hello world!')
>>> str(MyError2('hello world!'))
'hello world!'
```

Reply:
Likewise below

Reply:
Say it ain't so - just committed the changes